docs/conf.py

```diff
@@ -17,8 +17,8 @@
 
 html_context = {
   'version' : '3.1',
-  'full_version' : '3.1.0',
-  'maven_plugin_version' : '3.1.0-RC6',
+  'full_version' : '3.1.1',
+  'maven_plugin_version' : '3.1.0-RC7',
```

Comment:
Would be awesome to have a release version on the maven plugin, but no biggie :)

Reply:
This is actually in RC8 now, by the way...